docs/technical_reference/unit_models/pressure_exchanger.rst

```diff
@@ -32,6 +32,13 @@ Where the system is also subject to following constraints:
     
     Figure 1. Schematic representation of an energy recovery system using a pressure exchanger.
 
+When setting the ``has_leakage`` configuration option to ``True``, there is 1 additional variables ``leakage_vol`` that must be fixed.
```

Comment:
typo: "1 additional variables" should drop the plural s -> "1 additional variable"